packages/api-gateway/src/graphql-common-utils.ts

```diff
@@ -0,0 +1,84 @@
+import { Connection, cursorToOffset, offsetToCursor } from 'graphql-relay';
+import type { QueryInput } from '@aloxide/demux';
+
+const DEFAULT_ITEM_PER_PAGE = 20;
+
+export function updateCursorToOffet(args: any): QueryInput {
```

Comment:
There're a few issues with this function name, it's easily misleading, I need to read function body to know what it really does since I can't guess from its name:

- Update something often means we change its value, not change its structure/layout. Not making a new thing.
- To change something into another thing, you should use make, convert, create from etc.

Reply:
changed!